src/api/ReviewApi.js

```diff
@@ -1,11 +1,16 @@
 import axios from "axios";
 const BASE_URL = 'http://localhost:4000/api'
 
-export const getReviewByProductId = async (productId) => {
-    try {
-        const { data } = await axios.get(`${BASE_URL}/reviews/${productId}`);
-        return data;
-      } catch (error) {
-        throw error;
-    }
-    };
+
+//fetch review
+const fetchReview = async (productId) => {
+  axios
+    .get(`BASE_URL/reviews/${productId}`)
+    .then((res) => {
+      setRatings(res.data);
+    })
+    .catch((err) => {
+      console.log(err)
+    })
+  setIsLoading(false)
+}
```

Comment:
we want to format this request to just be the axios part of the request, similar to getReviewByProductId

we can then setRatings and setIsLoading back in the ReviewContext react component